shopinvader_search_engine_image/tests/common.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import base64
+import io
+
+from odoo_test_helper import FakeModelLoader
+from PIL import Image
+
+from odoo.addons.connector_search_engine.tests.common import TestSeBackendCaseBase
+from odoo.addons.extendable.tests.common import ExtendableMixin
+
+
+class TestSeMultiImageThumbnailCase(TestSeBackendCaseBase, ExtendableMixin):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.init_extendable_registry()
+        cls.loader = FakeModelLoader(cls.env, cls.__module__)
+        cls.loader.backup_registry()
+        from odoo.addons.connector_search_engine.tests.models import SeBackend, SeIndex
+
+        cls.loader.update_registry(
+            (
+                SeIndex,
+                SeBackend,
+            )
+        )
+        cls.backend = cls.env["se.backend"].create(
+            {"name": "Fake SE", "tech_name": "fake_se", "backend_type": "fake"}
+        )
```

Comment:
I don't know when tests ran on GitHub, but they fail for me today, when creating the backend:

```
 File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/addons/shopinvader_search_engine_image/tests/common.py", line 28, in setUpClass
    cls.backend = cls.env["se.backend"].create(
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/api.py", line 409, in _model_create_multi
    return create(self, [arg])
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/addons/server_environment/models/server_env_tech_name_mixin.py", line 64, in create
    return super().create(vals_list)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/api.py", line 410, in _model_create_multi
    return create(self, arg)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/addons/base/models/ir_fields.py", line 665, in create
    recs = super().create(vals_list)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/api.py", line 410, in _model_create_multi
    return create(self, arg)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/models.py", line 3967, in create
    next(iter(fields)).determine_inverse(batch_recs)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/fields.py", line 1377, in determine_inverse
    determine(self.inverse, records)
  File "/home/marie/.virtualenvs/odoo-shopinvader/lib/python3.10/site-packages/odoo/fields.py", line 96, in determine
    if needle.__name__.find('__'):
AttributeError: 'functools.partial' object has no attribute '__name__'

```